### DISCUSSION TITLE
Screenshots in Readme Update

### DISCUSSION THREAD

**Discussion by Kitcham** [kwsch/PKHeX#3645] [Ideas]
Since the translation has been completed, it may be possible to update the screenshots in the README_zhHK file. One small issue is that the language now used in the screenshots is Simplified Chinese (简体中文), not Traditional Chinese (繁體中文). It should be set to use Traditional Chinese.

Likewise, the screenshots in Spanish and Italian also could be updated.

**Comment by kwsch** [comment]
If you stage a save file screenshot (idc about OT shown, but so long as it is "clean") with a pull request hotlinking to an imgur upload, it'll be fine to merge.

The screenshot is provided mostly an example of what the program generally looks like, not what it currently looks like 😆

**Reply by Kitcham** [reply]
OK, I will create a PR for that later.

And THX for providing different screenshots for different languages this time~ 😉
